### DISCUSSION TITLE
Support for piped.video

### DISCUSSION THREAD

**Discussion by trup** [aclap-dev/video-downloadhelper#1117] [Help and Q&A]
### Your question

Hello.
Please add support for `https://piped.video`

### Addon & CoApp version and details

version: 9.1.0.42
target: mozilla
channel: stable
lang: ru
coapp: {"found":true,"path":"C:\\Program Files\\DownloadHelper CoApp\\vdhcoapp.exe","version":"2.0.19","new_version":false}
license: {"unset":true}
platform: x86-64 win
UA: Mozilla/5.0 (Windows NT 10.0; Win64; x64; rv:129.0) Gecko/20100101 Firefox/129.0
{
    "used_history_button": true,
    "download_directory": "C:\\Users\\AnnA\\Videos",
    "last_download_directory": {
        "__serializer_tag": "option_some",
        "__serializer_value": {
            "__serializer_tag": "primitive",
            "__serializer_value": "C:\\Users\\AnnA\\Videos"
        }
    },
    "concurrent_downloads_max": 5,
    "never_show_no_incognito_msg_again": true,
    "view_options": {
        "all_tabs": false,
        "low_quality": false,
        "sort_by_status": true,
        "sort_reverse": false,
        "show_button_clean": true,
        "show_button_clean_all": false,
        "show_button_convert_local": false,
        "hide_downloaded": true
    },
    "show_context_menu": false,
    "yt_warning": false,
    "use_wide_ui": false,
    "use_legacy_ui": false,
    "use_sidebar": false,
    "open_count_store": 4,
    "successfull_dl": 81,
    "record_download_history": false
}

**Comment by Mattmjs** [comment]
I suggested support for this on the old google groups forum in case YouTube changed something to break downloading.
But it seems there is a bug with every video with _Got error: "Sign in to confirm that you're not a bot"_   even when signed in.
It works at **https://piped.yt**
